Command/LoadDataFixturesDoctrineCommand.php

```diff
@@ -84,9 +90,28 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $em->getConnection()->connect($input->getOption('shard'));
         }
 
-        $fixtures = $this->fixturesLoader->getFixtures();
+        $dirOrFile = $input->getOption('fixtures');
+        if ($dirOrFile) {
+            $paths = is_array($dirOrFile) ? $dirOrFile : array($dirOrFile);
+            $loader = new DataFixturesLoader($this->getContainer());
```

Comment:
why don't you use the `$this->loader` instead ?

Reply:
because it always load all fixtures